src/test/kotlin/it/pagopa/wallet/controllers/WalletControllerTest.kt

```diff
@@ -93,19 +96,15 @@ class WalletControllerTest {
     @Test
     fun testCreateWallet() = runTest {
         /* preconditions */
-
-        given { walletService.createWallet(any(), any(), any(), any()) }
-            .willReturn(
-                mono {
-                    Pair(
-                        LoggedAction(
-                            WALLET_DOMAIN,
-                            WalletAddedEvent(WALLET_DOMAIN.id.value.toString())
-                        ),
-                        webviewPaymentUrl
-                    )
-                }
+        val pairLoggedActionUri =
+            Pair(
+                LoggedAction(WALLET_DOMAIN, WalletAddedEvent(WALLET_DOMAIN.id.value.toString())),
+                webviewPaymentUrl
             )
+        given { walletService.createWallet(any(), any(), any(), any()) }
```

Comment:
please add test verification stating that `walletEventSinksService.tryEmitEvent` is called once with the expected results. without verification test will succeed even if the tryEmitEvent method call is removed from the controller or if the passed parameter arguments are not the expected ones

Reply:
this comment applies to all below test cases

Reply:
Done in [a18f90a](https://github.com/pagopa/pagopa-wallet-service/pull/195/commits/a18f90a83a44c00163fc07e09c48a4c586e00661)